bundle.Dockerfile

```diff
@@ -13,9 +13,9 @@ LABEL name="openshift4-wincw/windows-machine-config-operator-bundle" \
 # delivered via an index image
 LABEL com.redhat.delivery.operator.bundle=true
 
-# This second label tells the pipeline which versions of OpenShift the operator supports (4.6+).
+# This second label tells the pipeline which versions of OpenShift the operator supports (4.7).
 # This is used to control which index images should include this operator.
-LABEL com.redhat.openshift.versions="v4.6"
+LABEL com.redhat.openshift.versions="=v4.7"
```

Comment:
Why another `=`, is this a requirement from OLM team?

Source: https://coreos.slack.com/archives/C3VS0LV41/p1611074292076200

Reply:
The `=` means that it should only be present in the `v4.7` index